system-addon/content-src/components/SectionMenu/SectionMenu.jsx

```diff
@@ -5,12 +5,13 @@ import React from "react";
 import {SectionMenuOptions} from "content-src/lib/section-menu-options";
 
 const DEFAULT_SECTION_MENU_OPTIONS = ["MoveUp", "MoveDown", "Separator", "RemoveSection", "CheckCollapsed", "Separator", "ManageSection"];
+const WEBEXT_SECTION_MENU_OPTIONS = ["MoveUp", "MoveDown", "Separator", "CheckCollapsed", "Separator", "ManageWebExtension"];
 
 export class _SectionMenu extends React.PureComponent {
   getOptions() {
     const {props} = this;
 
-    const propOptions = Array.from(DEFAULT_SECTION_MENU_OPTIONS);
+    const propOptions = props.isWebExtension ? [...WEBEXT_SECTION_MENU_OPTIONS] : [...DEFAULT_SECTION_MENU_OPTIONS];
     // Prepend custom options and a separator
     if (props.extraOptions) {
```

Comment:
I suppose we don't want Manage Extension to be the first option separated? I don't see a spec, but this removing of "remove" and replacing "manage section" seems reasonable.